rackcity/api/objects/rack_range.py

```diff
@@ -1,8 +1,13 @@
 from rest_framework import serializers
 from rackcity.models.rack import validate_row_letter
+from rackcity.models import Datacenter
 
 
 class RackRangeSerializer(serializers.Serializer):
+    datacenter = serializers.PrimaryKeyRelatedField(
+        label='datacenter',
+        queryset=Datacenter.objects.all()
+    )
```

Comment:
if you're going to add this to the rack range serializer, i think it should be renamed to something like rack filter serializer, because datacenter is not part of a rack range